apps/condo/pages/_app.tsx

```diff
@@ -172,18 +186,23 @@ const MyApp = ({ Component, pageProps }) => {
                         <TrackingProvider>
                             <OnBoardingProvider>
                                 <SubscriptionProvider>
-                                    <LayoutContextProvider>
-                                        <LayoutComponent menuData={<MenuItems/>} headerAction={HeaderAction}>
-                                            <RequiredAccess>
-                                                <Component {...pageProps} />
-                                                {
-                                                    isEndTrialSubscriptionReminderPopupVisible && (
-                                                        <EndTrialSubscriptionReminderPopup/>
-                                                    )
-                                                }
-                                            </RequiredAccess>
-                                        </LayoutComponent>
-                                    </LayoutContextProvider>
+                                    <TasksContextProvider
+                                        initialTaskRecords={initialTaskRecords}
+                                        uiInterfaces={{ MiniAppTask: MiniAppTaskUIInterface }}
```

Comment:
please memoize this or it will cause redundant rerenders of it's children

Reply:
done